### ISSUE TITLE
how to use this library

### ISSUE THREAD

**Issue by itconsultore** [eugeniobonifacio/xmldsig-xades#1]
do you have any sample of how to use this xades to sign XML
thanks

**Comment by eugeniobonifacio** [comment]
Take a look at src/XAdES.php

**If you want to verify a signature**

$xades = new XAdES(null, null); // you don't need keys for signature verification, it could be done better with a default constructor

$data = new SignatureData();
$data->setName('document.xml');
$data->setContent($xml); // $xml is the signed XML document string

$result = $xades->verify($data); // true if signature is valid, otherwise false


**If you want to sign a document** 


$xades = new XAdES($pubKey, $pvtKey); // You need both keys to sign

$data = new SignatureData();
$data->setName('document.xml');
$data->setContent($xml); // $xml is the plain XML document string to be signed

$signed = $xades->sign($data);

$signedXML = $signed->getContent(); // $signedXML contains the signed XML document string

I hope this can help you.

**Comment by itconsultore** [comment]
Great.

I understand that the private key is a string (text plain).
What about the public key? I have a .p12 file, should I get the content as byte and use it as $pubKey?
I tried with a URL and use the $isURL=True but it didn't work.
I exported the certificate as a .cer file and put its content as base64 string for public key, but it didn't work neigher.
any other suggestions?

I have this error:
Failure Signing Data: error:0909006C:PEM routines:get_name:no start line - SHA256

Thank you.
